src/client/components/filter-menu/boolean-filter-menu/boolean-filter-menu.scss

```diff
@@ -17,8 +17,8 @@
 
 @import '../../../imports';
 
-$checkbox-width: 20px;
 $table-height: 80px;
+$label-height: 24px;
```

Comment:
Zmienna ta powtarza się w sumie w dwóch plikach, nie dałoby się np. przenieść do innego arkusza, który importował by one?

Reply:
Let me find good place for it in the depths of sass imports ...